src/06-blackjack/Blackjack/Blackjack.Core/Deck.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Blackjack.Core
+{
+    public class Deck
+    {
+        private readonly List<Card> _cards = new List<Card>(52);
+        private string[] _suits = new string[] { "♠", "♥", "♣", "♦" };
+        private string[] _ranks = new string[] { "A","2","3","4","5","6","7","8","9","10","J","Q","K" };
+
+        public Deck()
+        {
+            AddCards();
+        }
+
+        public void AddCards()
+        {
+            foreach (var suit in _suits)
+            {
+                foreach (var rank in _ranks)
+                {
+                    if (rank == "J"|| rank == "Q" || rank == "K")
+                    {
+                        this._cards.Add(new Card(rank.ToString(),suit,10));
+                    }
+                    else if (rank == "A")
+                    {
+                        //The Ace is always valued at 11. When hand going over 21, Ace is valued as 1.
+                        this._cards.Add(new Card(rank.ToString(), suit, 11));
+                    }
+                    else
+                    {
+                        this._cards.Add(new Card(rank, suit, int.Parse(rank)));
```

Comment:
This line seems wierd... rank is a string... you are doing `int.Parse(string)`

It might be doing what you expect (points), but this is really confusing to read.. Why would I need to int.Parse a rank?